### ISSUE TITLE
lower accuracy on OMD(Obama-McCain Debate twitter sentiment dataset)

### ISSUE THREAD

**Issue by AIRobotZhang** [huggingface/pytorch-pretrained-BERT#133]
I run the classification task with BERT pretrianed model, but while it's much lower than other methods on OMD dataset, which has 2 labels. The final accuracy result is only 62% on binary classification task!

**Comment by rodgzilla** [comment]
We need more informations on the parameters you use to run this training in order to understand what might be wrong.

**Comment by AIRobotZhang** [comment]
> We need more informations on the parameters you use to run this training in order to understand what might be wrong.

THANK YOU! Because of limited mermory, the batch_size is 8 and epoch is 6, and the content is short, so set the max_length is 50, other parameters are default.

**Comment by thomwolf** [comment]
Try various values for the hyper-parameters and at least 10 different seed values.
Limited memory should not be a limitation when you use `gradient accumulation` as indicated in the  readme [here](https://github.com/huggingface/pytorch-pretrained-BERT#training-large-models-introduction-tools-and-examples) (see also how it is used in all the examples like `run_classifier`, `run_squad`...)
